homeassistant/components/snoo/services.yaml

```diff
@@ -0,0 +1,27 @@
+# Describes the format for available Snoo services
+log_diaper_change:
+  fields:
+    baby_id:
+      required: true
+      example: "35235211235dfasdf32523"
+      selector:
+        text:
+    diaper_types:
+      required: true
+      example: ["Wet", "Dirty"]
+      selector:
+        select:
+          multiple: true
+          options:
+            - "Wet"
+            - "Dirty"
+    note:
+      required: false
+      example: "Wet diaper"
```

Comment:
Can we pick a better example because the diaper types already indicate if it was Wet

Reply:
So - tried to use a translation key here, e.g. `[%key:component::snoo::services::log_diaper_change::fields::diaper_types::options::dirty%]` but was showing up as that literal string. Was I doing that wrong or did you mean something else?